images/hub/Dockerfile

```diff
@@ -56,10 +53,16 @@ RUN apt-get update && \
         tini \
  && rm -rf /var/lib/apt/lists/*
 
-COPY --from=build-stage /build-stage/*.whl /tmp/pre-built-wheels/
+# set env for pip cache,
+# but use ARG to avoid persisting in image
+ARG PIP_CACHE_DIR=/tmp/pip-cache
```

Comment:
Ah I verified that to be so, nice! I didn't know this. I also wasn't sure that PIP_CACHE_DIR was respected by PIP itself or something we gave meaning to. This variation of the comment could have perhaps helped med understand this a bit quicker.

```suggestion
# set pip's cache directory with environment variable and use
# ARG instead of ENV to ensure its only set when the image is built
ARG PIP_CACHE_DIR=/tmp/pip-cache
```